api/models/v0.2/modelResponse.js

```diff
@@ -10,26 +10,18 @@ const url = require('url');
  */
 function modelPatronCreatorResponse(responseData, status, requestData) {
   return {
-    data: [
```

Comment:
We are intentionally removing the containing array so that we only return one patron record.

Reply:
Do we know which patron record gets chosen if there's more than one?

Reply:
@daryanypl Sierra will respond with the one patron we created, even if there are others with duplicate emails or other data.